superset/utils/dashboard_import_export.py

```diff
@@ -22,6 +22,16 @@
 from superset.models.core import Dashboard
 from superset.utils.core import decode_dashboards
 
+def dashboard_json(data):
+    return json.loads(data, object_hook=decode_dashboards)
+
+def import_dashboard_json(session, data, import_time=None):
+    current_tt = int(time.time())
+    import_time = current_tt if import_time is None else import_time
+    dashboard = data['dashboards'][0]
```

Comment:
is there any key with which we can retrieve the value here or has it to be on 0th index always?

Reply:
This is array we can iterate but as mentioned we are only supporting single dashboard at this point of time.

There will more changes than just iteration we need to take care (rest API signature will change). Hence will differ it till a ask come or we hit a persuasive usecase